src/interface/substrate_query.js

```diff
@@ -428,13 +459,34 @@ module.exports = {
                 } catch (e) {
                     return {'error': e.toString()};
                 }
+	    case 'staking/payee':
+                try {
+		    if (!param2) {
+                        return {'error': 'You did not enter the stash '
+                                + 'address that needs to be queried'};
+                    }
```

Comment:
same thing here, kindly check that the `}` and the `return` are aligned with the `if` above